### PR TITLE
Finished Input and Lists components

### PR DESCRIPTION
**Pull request by Naman-Saxena1**

### CODE REVIEW

File: Components/Input/form-input.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+formSubmitBtn.addEventListener('click',()=>{
+    let emailFormatErrorFlag = false;
+    for(index in formInputEmail.value)
+    {
+        if(!(formInputEmail.value[index]=="@"))
+        {
+            emailFormatErrorFlag=true;
+        }
+
```
Comment (Ankur9669) [inline Components/Input/form-input.js:14]: You can try using regex here. But this is also fine for now👍.

File: Components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    display: flex;
+    flex-direction: column;
+    background-color: #F5F7F9;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 2rem;
+    padding: 4.8rem 1.6rem;
+}
+
```
Comment (Ankur9669) [inline Components/Input/input.css:15]: I think you have not used input container class anywhere, if it is unused please remove it

File: Components/Lists/lists.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.post-options button
+{
+    min-width: 0;
+    flex-grow: 1;
+}
+
+
+
```
Comment (Ankur9669) [inline Components/Lists/lists.css]: These white spaces could be removed

File: Components/Lists/lists.html [hunk 4]
```diff
@@ -75,13 +76,223 @@
...
+                                        <h2>Jane</h2>
+                                    </div>
+                                    <button class="icon-btn"><i class="fa fa-ellipsis-h fa-x" aria-hidden="true"></i></button>
+                                </div>
+                                <hr>
+                                <p class="post-caption">
+                                    My new setup!
+                                </p>
+                                <img class="post-img" src="../../Icons-and-Images/List-Images/max-andrey-qdhjaVAOhbY-unsplash.jpg" alt="cafe-img">
+                                <div class="post-stats">
```
Comment (Ankur9669) [inline Components/Lists/lists.html:182]: img tag does not have closing /

**Review by Ankur9669 — APPROVED** [review]
The code is looking great. Just one suggestion in lists you can reduce the height of images it is very long.

File: Components/Input/form-input.js (hunk 1 above)
Comment (Naman-Saxena1) [reply Components/Input/form-input.js:14]: Ok, will check it out.

File: Components/Input/input.css (hunk 2 above)
Comment (Naman-Saxena1) [reply Components/Input/input.css:15]: Done.

File: Components/Lists/lists.html (hunk 4 above)
Comment (Naman-Saxena1) [reply Components/Lists/lists.html:182]: Done.

File: Components/Lists/lists.css (hunk 3 above)
Comment (Naman-Saxena1) [reply Components/Lists/lists.css]: Done.

File: Components/Lists/lists.css [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
+}
+
+.post-stats-left-container i
+{
+    color: white;
+    padding: 1rem;
+    background-color: #3b5998;
```
Comment (bhtibrewal) [inline Components/Lists/lists.css:121]: use variable instead of color name

File: Components/Lists/lists.css [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.post-stats-left-container
+{
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
+}
+
+.post-stats-left-container i
```
Comment (bhtibrewal) [inline Components/Lists/lists.css:117]: add cursor pointer on the like button

File: Components/Lists/lists.css [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    display: flex;
+    padding: 0 5px;
+    justify-content: space-between;
+}
+
+.post-stats-left-container
+{
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
```
Comment (bhtibrewal) [inline Components/Lists/lists.css:114]: padding is a little less from my perspective, and you should just keep two buttons there 3 with same significance confuses the user

**Review by bhtibrewal — APPROVED** [review]
I have listed a few minor points please look into them and there is no effect on the submit button in the input component.
everything else looks good to me, good work there

File: Components/Lists/lists.css (hunk 7 above)
Comment (Naman-Saxena1) [reply Components/Lists/lists.css:114]: Thanks, will look into it.

File: Components/Lists/lists.css (hunk 6 above)
Comment (Naman-Saxena1) [reply Components/Lists/lists.css:117]: Yeah, cursor pointer is already present on all buttons, I have put it universally for all buttons, you can check on preview.

File: Components/Lists/lists.css (hunk 5 above)
Comment (Naman-Saxena1) [reply Components/Lists/lists.css:121]: Done.

**Pull request merged**